CIScripts/Test/TestConfigurationUtils.ps1

```diff
@@ -473,7 +474,24 @@ function New-Container {
     if ($Name) { $Arguments += "--name", $Name }
     $Arguments += "--network", $NetworkName, $Image
 
-    $ContainerID = Invoke-Command -Session $Session { docker @Using:Arguments }
+    $Result = Invoke-NativeCommand -Session $Session -CaptureOutput -AllowNonZero { docker @Using:Arguments }
+    $ContainerID = $Result.Output.Split([Environment]::NewLine) | Select-Object -First 1
+    $OutputMessages = $Result.Output
+
+    # Workaround for occasional failures of container creation in Docker for Windows.
```

Comment:
Just two comments:
* can we add this to HNS bestiary?
* maybe we should move comment + workaround code to separate function(s)?